modules/util/portal-tools-soy-builder/src/main/resources/META-INF/maven/plugin.xml

```diff
@@ -43,7 +43,7 @@
 				</parameter>
 			</parameters>
 			<configuration>
-				<_baseDir default-value="${project.basedir}" implementation="java.io.File"/>
```

Comment:
@Seiphon were the changes to this file auto-generated?

Reply:
> @Seiphon were the changes to this file auto-generated?

Yes, this changes are generated after executing "gw installCache"

Reply:
Great, thanks for the clarification!